### PR TITLE
Feat/sanity integration

### PR DESCRIPTION
**Pull request by wirayuda299**
Hi @brandonetter 

here is list what i do in this branch

- Create schema for sanity
- Integrate sanity
- Create copy text function
- I make different layout with this web and sanity studio
- Update some styling

### CODE REVIEW

File: utils/caseStudies.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+		throw error;
+	}
+};
+
+export const getSimilarCaseStudies = async (id: string) => {
+	try {
+		const diffProject = await client.fetch(
+			`*[_type == "caseStudies" && _id != "${id}"][0..1]`
+		);
+
```
Comment (brandonetter) [inline utils/caseStudies.ts]:
```suggestion
	try {
		const diffProject = await client.fetch(
			`*[_type == "caseStudies" && _id != $id][0..1]`,{id}
		);
```
You can pass params to sanity by giving it a 'params' object as the second argument

File: utils/caseStudies.ts
Comment (brandonetter) [inline utils/caseStudies.ts:1]: You could consider moving all of your Sanity queries into the sanity folder to keep things organized.

Perhaps an `actions` or `queries` folder with files named after each resource (like you already have).

**Review by brandonetter — CHANGES_REQUESTED** [review]
image

The idea from the figma is that hovering over the section will color the icon. Not that only the firefox logo is colored.

You also have a weird color choice of making some of these sections pitch black. Please fix that to match the design.

File: utils/caseStudies.ts
Comment (wirayuda299) [reply utils/caseStudies.ts:1]: Thanks for your feedback🙏

**Pull request merged**